bin/stack-helpers.sh

```diff
@@ -1,11 +1,12 @@
-set -o pipefail
+# shellcheck shell=bash
 
 function display() {
-  echo -e "\n----->" $*
+  echo -e "\n-----> $1"
```

Comment:
What is the story with this change?

Reply:
Ah so shellcheck warns on usage of unquoted `$*` since it applies both word splitting and globbing:
https://github.com/koalaman/shellcheck/wiki/SC2048

The error could have been resolved by changing that line to:
```sh
   echo -e "\n-----> $*"
```

...however it seemed cleaner to have the API of `display` and `abort` be that they are passed quoted strings, to avoid issues with word splitting, and also so that it's clearer when viewing in an editor with syntax highlighting, that they are message strings and not arguments to a command.

Happy to change it back if preferred :-)

Reply:
That is totally fine. I just noticed it _looked_ like it would change behavior. As long as we are confident we didn't use it in that way I'm 👍  cleaning up the API here to provide some safety and symmetry.